### ISSUE TITLE
yapf changes placement of line break near dot based on line length

### ISSUE THREAD

**Issue by SaintWacko** [google/yapf#806]
yapf seems to decide whether to break before or after a dot on chained calls based on line length. While I can understand why it does that, it leaves the code looking very inconsistent.

**Comment by gwelymernans** [comment]
I agree. yapf tries not to split around dots, but sometimes has to. Do you have an example that I can work with?

**Comment by SaintWacko** [comment]
I've just been messing with this:
`def test():
    LOGGER.info(
        'GENERATOR - CONTENT - ORG {0} - [ContentQuestion] -- header builtasd'.
        format(org_id))`

Obviously that won't compile, but yapf doesn't care about that. If you add one character to the string, it'll kick the dot down to the next line (which is where I prefer it)

**Comment by gwelymernans** [comment]
What do you mean it won't compile?  It's well-formed and the parser seems happy with it. Try setting `SPLIT_BEFORE_DOT` to `True` and see if that helps:

```
$ python3.7 -m yapf --style '{based_on_style: pep8, split_before_dot: True}' < /tmp/x.py
def test():
    LOGGER.info(
        'GENERATOR - CONTENT - ORG {0} - [ContentQuestion] -- header builtasd'
        .format(org_id))
```
